packages/language-server/src/__test__/completion.test.ts

```diff
@@ -1956,6 +1956,31 @@ suite('Completions', function () {
         },
       })
     })
+    test('enum UserType |', () => {
```

Comment:
maybe split into two tests, so we have an explicit one that only tests that commented out @id does not disable the completion?